081-unregistration-of-KRaft-nodes.md

```diff
@@ -0,0 +1,117 @@
+# Unregistration of KRaft nodes
+
+This proposal discusses the support for KRaft node unregistration when removing the nodes.
+This is one of the missing features in Strimzi's KRaft support.
+
+## Current situation
+
+When removing Apache Kafka nodes from a KRaft-based cluster, the nodes should be unregistered using the Apache Kafka Admin API.
+When the nodes are not properly unregistered, the cluster will keep tracking them and they would cause problems later.
+For example, when you upgrade your cluster to a new Kafka minor version and try to also bump the metadata version, the metadata version update will be refused because the removed nodes that are still registered with the old Kafka version and do not support the new metadata version.
```

Comment:
```suggestion
For example, when you upgrade your cluster to a new Kafka minor version and try to also bump the metadata version, the metadata version update is refused because the removed nodes are still registered with the old Kafka version and do not support the new metadata version.
```